sticky_notes.py

```diff
@@ -18,23 +35,21 @@ def stickynote(self, event):
         elif len(args) < 2:
             return "Sticky note needs to contain some actual note."
         else:
-            user = event.args[0]
-            content = ' '.join(event.args[1:])
+            to_nick = event.args[0]
+            from_nick = event.meta['nick']
+            msg = ' '.join(event.args[1:])
```

Comment:
@JakubNvk if you prefer `"` over `'` why don't you use them here?

I'd personally use `'` for such a simple strings as these (and git blame will show that I am indeed the original author of this line) but if you want to use `"` the most important thing is to be consistent.

That being said, either replace `'` with `"` here (and on the line above this one) or change the other simple strings (like `sender = ""`) to '`' since that is what seems to be convetion throughout this file.